official/vision/beta/projects/yolo/losses/yolo_loss.py

```diff
@@ -111,8 +111,8 @@ def __init__(self,
         max_delta=self._max_delta)
     self._decode_boxes = functools.partial(
         loss_utils.get_predicted_box, **box_kwargs)
-    self._search_pairs = lambda pred_boxes, pred_classes, boxes, classes, scale, yxyx: (None, None, None, None)  # pylint:disable=line-too-long
-    self._build_per_path_attributes()
+
+    self._search_pairs = None
```

Comment:
You can't initialize this to None, the internal tests would fail since you have self._search_pairs(**kwargs) below.

Reply:
fixed